compliance/nvidia/TEST01/verify_accuracy.py

```diff
@@ -120,7 +120,9 @@ def main():
         print("num_perf_log_qsl_idx_match = {:}".format(num_perf_log_qsl_idx_match))
         print("num_perf_log_data_mismatch = {:}".format(num_perf_log_data_mismatch))
         print("num_missing_qsl_idxs = {:}".format(num_missing_qsl_idxs))
-        if num_perf_log_data_mismatch > 0 :
+        # Check if there are a mismatches or queries are missing from the reference 
+        # accuracy or the test accuracy is not empty, fail the test in that case
+        if num_perf_log_data_mismatch > 0 or num_missing_qsl_idxs > 0 or len(perf_data) == 0:
```

Comment:
Hi @pgmpablo157321 this looks like a more stringent check and may cause unintended failures. I think [this fix](https://github.com/mlcommons/submissions_inference_v3.1/issues/92#issuecomment-1701774279) from Victor might be a better choice.